cli/image-scanning-by-using-the-roxctl-cli.adoc

```diff
@@ -0,0 +1,71 @@
+[id="image-scanning-by-using-the-roxctl-cli"]
+:_content-type: ASSEMBLY
+= Image scanning by using the roxctl CLI
+include::modules/common-attributes.adoc[]
+:context: image-scanning-by-using-the-roxctl-cli
+
+toc::[]
+
+[role="_abstract"]
+You can use `roxctl` CLI, Jenkins, and API to scan images stored in a cluster specific {ocp} integrated image registry. To do so, specify the appropriate cluster using the delegated scanning configuration or the cluster parameter in `roxctl` CLI, Jenkins, and API.
+
+[id="scanning-images-in-a-remote-cluster_{context}"]
+== Scanning images in a remote cluster
+
+To scan an image in a remote cluster, use the `roxctl image scan` command. Specify the location and name of the image, and specify the remote cluster. You can also specify optional parameters to customize the behavior of the command.
+
+.Procedure
+
+* Run the following command to scan the specified image in a remote cluster:
++
+[source,terminal,subs="+quotes"]
+----
+$ roxctl image scan --image=_<image_registry>_/_<image_name>_ --cluster=_<cluster_name>_ [flags]<1>
+----
++
+--
+<1> For `<image_registry>`, specify the registry where the image is located, for example, `image-registry.openshift-image-registry.svc:5000/`.
++
+For `<image_name>`, specify the name of the image you want to scan, for example, `dave/dave-is:latest`.
++
+For `<cluster_name>`, specify the name of the remote cluster, for example, `remote`.
++
+Optional: For `[flags]`, you can specify parameters to modify the behavior of the command.
++
+For more information about optional parameters, see xref:../cli/image-scanning-by-using-the-roxctl-cli.adoc#roxctl-image-scan-command-options_image-scanning-by-using-the-roxctl-cli[roxctl image scan command options].
+--
++
+.Example output
++
+[source,text,subs=attributes+]
+----
+{
+  "Id": "sha256:3f439d7d71adb0a0c8e05257c091236ab00c6343bc44388d091450ff58664bf9", <1>
+  "name": { <2>
+    "registry": "image-registry.openshift-image-registry.svc:5000", <3>
+    "remote": "dave/dave-is", <4>
+    "tag": "latest", <5>
+    "fullName": "image-registry.openshift-image-registry.svc:5000/dave/dave-is:latest" <6>
+  },
+  "names": [ <7>
+    {
+      "registry": "image-registry.openshift-image-registry.svc:5000",
+      "remote": "dave/dave-is",
+      "tag": "latest",
+      "fullName": "image-registry.openshift-image-registry.svc:5000/dave/dave-is:latest"
+    }
+  ],
```

Comment:
Recommend removing `names` from the output and the `#7` bullet, the `names` field is more for internal purposes at this time.